Makefile

```diff
@@ -72,3 +72,8 @@ taint: ## Taint a resource, make taint project=<project name> resource=<resource
 .PHONY: jump
 jump: ## Jump onto the first instance on a dev environment
 	. ./setup.sh -j
+
+
+.PHONY: create_console
+create_console: ## Create console session
```

Comment:
Our convention appears to be using `-` instead of `_` for the task name.

Also please could we include an example to our description i.e. `Create console session, make create-console project=<project name>`